plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -647,13 +655,24 @@ private class DokkaDescriptorVisitor(
         return coroutineScope {
             val generics = async { descriptor.typeParameters.parallelMap { it.toVariantTypeParameter() } }
 
+            fun SourceSetDependent<DocumentationNode>.translateParamToDescription(): SourceSetDependent<DocumentationNode> {
+                return this.mapValues { (_, value) ->
+                    value.copy(children = value.children.map {
+                        when (it) {
+                            is Property -> Description(it.root)
+                            else -> it
+                        }
+                    })
+                }
+            }
```

Comment:
It's a little hacky, though it makes no sense that inheriting getter/setter has `Property` wrapper from the description of the property in classlike kdoc header.

Reply:
Done